build.gradle

```diff
@@ -41,32 +67,25 @@ sourceSets {
             srcDirs = ['src/resources']
         }
     }
+    test {
+        java {
+            srcDirs = ['test']
+        }
+        resources {
+            srcDirs = ['test/resources']
+        }
+    }
 }
 
 // Appending a project version to the jar. version is passed in with -PprodVersion=...
 allprojects {
-   if (project.hasProperty('prodVersion')) {
-     project.version = project.prodVersion
-   } else {
-     project.version = 'developer-build'
-   }
-}
-
-// Defining the jar manifest
-def robotManifest = {
-  attributes 'Main-Class': 'edu.wpi.first.wpilibj.RobotBase'
-  attributes 'Implementation-Title': 'Crate Robot'
-}
-
-task fatjar( type: Jar ) {
```

Comment:
why has the fatjar task been deleted? I think we should keep it in even with the gradle rio deploy artifact